src/scss/styles.scss

```diff
@@ -1,4 +1,7 @@
 @import "../../jslib/angular/src/scss/webfonts.css";
+$dark-icon-themes: 'theme_dark';
+@import "../../jslib/angular/src/scss/icons.scss";
```

Comment:
And this needs to be placed below fontawesome, since we override one of the css properties.